locales/sr/notes.properties

```diff
@@ -1,5 +1,45 @@
-welcomeTitle2=Здраво!
-welcomeText2=Добродошли на једностранични нотес уграђен у Firefox. Сурфујте вебом. Запишите белешке. Једноставно је.
+welcomeTitle3=Добродошли у Firefox Notes!
```

Comment:
Uhm, why do you have change to the Serbian localization in your PR?

Reply:
Serbian modifications come from a commit done yesteday on master ...  I don't understand why my PR see a diff here with master since on master I have same version ...

Reply:
Wild guess: at this point your PR touches the file twice, so it's like a standard modification that was committed only on the branch. I think you have to manually restore the file from master and commit to the branch.

Note that there are a few other conflicts against master (but not the Serbian file).